build/js/main.js

```diff
@@ -1034,6 +1047,11 @@ class Bluetooth extends Plugin {
         <div id="statusMessages" class="text grid__col grid__col--8-of-8"></div>
         `;
 
+        // Disable buttons
+        document.getElementById('BT_Pair').disabled = true;
```

Comment:
For readability and consistency its better if you assign these in a this.xyzButton value, like the this.connectButton and this.disconnectButton cases below. 

You can se the disable true directly in the template or at the same place as where you bind the buttons